src/axom/slic/interface/c_fortran/typesSLIC.h

```diff
@@ -14,15 +14,23 @@
 extern "C" {
 #endif
 
+// helper capsule_SLIC_GenericOutputStream
+struct s_SLIC_GenericOutputStream
+{
+  void *addr; /* address of C++ memory */
+  int idtor;  /* index of destructor */
+};
+typedef struct s_SLIC_GenericOutputStream SLIC_GenericOutputStream;
+
 // helper capsule_data_helper
 struct s_SLIC_SHROUD_capsule_data
 {
-  void* addr; /* address of C++ memory */
+  void *addr; /* address of C++ memory */
```

Comment:
These formatting changes strike me as odd since clangformat should convert it to `void*` from the Shroud generated `void *`. I edited the file to use `void*` then clangformat moved it back. Maybe we're treating C and C++ files differently.

This is more of a general problem and should be fixed in another PR.